src/cypress-api/index.ts

```diff
@@ -0,0 +1,65 @@
+import type { elementNode } from '@chromaui/rrweb-snapshot';
+import { writeTestResult } from '../write-archive';
+import { SourceMapper } from '../utils/source-mapper';
+import type { ChromaticStorybookParameters } from '../types';
+import type { ResourceArchive } from '../resource-archive';
+
+interface ArchiveParams {
+  testTitle: string;
+  domSnapshots: elementNode[];
+  resourceArchive: ResourceArchive;
+  chromaticStorybookParams: ChromaticStorybookParameters;
+}
+
+const doArchive = async ({
+  testTitle,
+  domSnapshots,
+  resourceArchive,
+  chromaticStorybookParams,
+}: ArchiveParams) => {
+  let sourceMap = null;
+
+  if (domSnapshots.length > 0) {
+    // shortens file names in the last snapshot (which is the automatic one)
+    const sourceMapper = new SourceMapper(domSnapshots[domSnapshots.length - 1]);
+    sourceMap = sourceMapper.shortenFileNamesLongerThan(250).build();
+  }
```

Comment:
This code will be obsolete once the source[-map-removal](https://github.com/chromaui/test-archiver/pull/28) PR is merged. This was just to make sure the final (automatic) snapshot had the correct truncated filenames.